sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -713,6 +713,8 @@ class StreamExecution(
 }
 
 object StreamExecution {
+  val OFFSET_LOG_VERSION = "v1"
```

Comment:
I removed this from StreamExecution. I was initially copying the approach in CompactibleStreamLog, but realize that the notion of a version need not be parameterized here. I'm still retaining the notion of a version, which no resides in the OffsetSeqLog companion object, as per your suggestion.